.vsts/python-setup.yml

```diff
@@ -0,0 +1,29 @@
+steps:
+  - task: UsePythonVersion@0
+    condition: eq( variables['Agent.OS'], 'Windows_NT' )
+    inputs:
+      versionSpec: '3.8.x'
+    displayName: Install Python 3.8 for Batch Explorer
+
+  - bash: ./scripts/azpipelines/setup-python.sh
+    displayName: Set up Python environment and private feed
+
+  - task: PipAuthenticate@1
+    displayName: 'Pip Authenticate'
+    inputs:
+      artifactFeeds: 'BatchExplorer'
+
+  - script: |
+      set -e
+      . "$(Agent.WorkFolder)/.venv/batchexplorer/bin/activate"
+      pip install -r python/requirements.txt
+    condition: ne( variables['Agent.OS'], 'Windows_NT' )
+    displayName: Install Python dependencies
```

Comment:
Super minor, but both of these always show up in the list of steps, so we may want to name them something like "Install Python dependencies (Windows)" & "Install Python dependencies (Linux)"

Reply:
Noticed that. Makes sense. Updated.